app/src/main/java/ru/yandex/practicum/contacts/presentation/filter/model/FilterContactTypeUi.java

```diff
@@ -12,6 +14,15 @@ public FilterContactTypeUi(@NonNull FilterContactType contactType, boolean selec
         this.selected = selected;
     }
 
+    @Override
+    public boolean theSameAs(FilterContactTypeUi item) {
+        if(this.getContactType() == item.getContactType()){
+            return true;
+        }else{
+            return false;
+        }
```

Comment:
Правильно, но лучше написать короче:
```suggestion
        return this.getContactType() == item.getContactType();
```
В других классах аналогично